src/Resource/GetAssets.php

```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
+
+namespace PayCrypto\Resource;
+
+use PayCrypto\CryptoMonitor;
+use PayCrypto\Collection\AssetsRecord;
+use ScriptFUSION\Porter\Connector\ImportConnector;
+use ScriptFUSION\Porter\Provider\Patreon\Collection\PledgeRecords;
+use ScriptFUSION\Porter\Provider\Patreon\PatreonProvider;
+use ScriptFUSION\Porter\Provider\Resource\ProviderResource;
+
+class GetAssets implements ProviderResource
+{
+    private $apiKey;
+
+    public function __construct(string $apiKey)
```

Comment:
It is bad design to specify the API key in the resource because it must be duplicated into every resource. Specify it in the connector instead, to eliminate this duplication.

Reply:
Should I set the encrypted env variable in ```.travis.yml``` file?

Thanks.

Reply:
Ok, I also use the ```travis-encrypt``` tool to encrypt the env variable.